NGitLab.Mock/Project.cs

```diff
@@ -149,6 +149,8 @@ public string[] Tags
 
         public ProtectedBranchCollection ProtectedBranches { get; }
 
+        public string RunnersToken { get; set; }
```

Comment:
I don't think you need a setter. The value should be generated in the ctor and I don't think it can be changed.

If you actually need a setter, maybe it could be `internal`

```suggestion
        public string RunnersToken { get; }
```

Reply:
I need a setter because when you instantiate a new project, the server is still null. We have to generate the project's token when we link it to the group/server collection